core/src/main/java/org/testcontainers/utility/ResourceReaper.java

```diff
@@ -127,49 +130,50 @@ private void stopContainer(String containerId, String imageName) {
     /**
      * Register a network to be cleaned up at JVM shutdown.
      *
-     * @param networkName   the image name of the network
+     * @param id   the ID of the network
      */
-    public void registerNetworkForCleanup(String networkName) {
```

Comment:
Since this method was semi-public, maybe it worth keep the old one and mark as deprecated, at least until 1.5.0?